fang/Cargo.toml

```diff
@@ -1,36 +1,42 @@
 [package]
 name = "fang"
-version = "0.10.4"
+version = "1.0.0"
 authors = ["Ayrat Badykov <ayratin555@gmail.com>" , "Pepe Márquez <pepe.marquezromero@gmail.com>"]
 description = "Background job processing library for Rust"
 repository = "https://github.com/ayrat555/fang"
 edition = "2021"
 license = "MIT"
 readme = "README.md"
-rust-version = "1.62"
+rust-version = "1.77"
```

Comment:
Recent command injection vulnerability is fixed in Rust 1.77.2

The reason of this is because soon I will require the last version of Rust to use delete `async recursion` crate and delete `async trait` crate